content/en/blog/_posts/2023-08-04-kubernetes-1.28-blog.md

```diff
@@ -0,0 +1,299 @@
+    
+The Kubernetes yearly support period already makes annual upgrades possible. Users can upgrade to the latest patch versions to pick up security fixes and do 3 sequential minor version upgrades once a year to "catch up" to the latest supported minor version.
+
+However, since the tested/supported skew between nodes and control planes is currently limited to 2 versions, a 3-version upgrade would have to update nodes twice to stay within the supported skew.
+
+## Beta support for enabling swap space on Linux
+    
+This adds swap support to nodes in a controlled, predictable manner so that Kubernetes users can perform testing and provide data to continue building cluster capabilities on top of swap.
+
+There are two distinct types of users for swap, who may overlap:
+
+- Node administrators, who may want swap available for node-level performance tuning and stability/reducing noisy neighbor issues.
+    
+- Application developers, who have written applications that would benefit from using swap memory. There are hence a number of possible ways that one could envision swap use on a node.
+    
+## Mixed version proxy (alpha) {#mixed-version-proxy}
+
+When a cluster has multiple API servers at mixed versions (such as during an upgrade/downgrade or when runtime-config changes and a rollout happens), not every apiserver can serve every resource at every version.
+
+For Kubernetes v1.28, you can enable the _mixed version proxy_ within the API server's aggregation layer.
+The mixed version proxy finds requests that the local API server doesn't recognize but another API server
+inside the control plan is able to support. Having found a suitable peer, the aggregation layer proxies
+the request to a compatible API server; this is transparent from the client's perspective.
+    
+When an upgrade or downgrade is performed on a cluster, for some period of time the API servers
+within the control plane may be at differing versions; when that happens, different subsets of the
+API servers are able to serve different sets of built-in resources (different groups, versions, and resources
+are all possible). This new alpha mechanism lets you hide that skew from clients.
+  
+## Source code reorganization for control plane components
+
+Kubernetes contributors have begun to reorganize the code for the kube-apiserver to build on a new staging repository that consumes [k/apiserver](https://github.com/kubernetes/apiserver) but has a bigger, carefully chosen subset of the functionality of kube-apiserver such that it is reusable.
+
+This is a gradual reorganization; eventually there will be a new
+[k/generic-controlplane](https://github.com/kubernetes/generic-controlplane) Git repository
```

Comment:
We should remove this link if the repo doesn't exist yet

Reply:
Thanks done.